dask_expr/_collection.py

```diff
@@ -1170,6 +1170,44 @@ def to_timestamp(self, freq=None, how="start"):
     def isna(self):
         return new_collection(self.expr.isna())
 
+    def random_split(self, frac, random_state=None, shuffle=False):
+        """Pseudorandomly split dataframe into different pieces row-wise
+
+        Parameters
+        ----------
+        frac : list
+            List of floats that should sum to one.
+        random_state : int or np.random.RandomState
+            If int create a new RandomState with this as the seed.
+            Otherwise draw from the passed RandomState.
```

Comment:
What if `None`?

Reply:
same as with int, is passed to numpy. added to the docs